invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_login.html

```diff
@@ -140,16 +145,20 @@ <h2 class="ui small header">{{ _("My account") }}</h2>
             <div class="ui divider"></div>
           {% endif %}
 
-          <a role="menuitem" class="item" href="{{ url_for_security('logout') }}">
-            <i class="sign-out icon"></i>
+          <a role="menuitem" class="item" href="{{ url_for_security('logout') }}"
+            id="auth-button"
+          >
+            <i class="sign-out icon" id="auth-icon"></i>
```

Comment:
id is meant to be unique in html (same problem in a tag), please use class attribute instead.
In addition, I would check the accessibility evaluation for what you've just added, I am not sure if screen reader will understand the loader without some additional attributes

Reply:
I thought the same, but in this case, there's always a single "Login"/"Logout" button + icon displayed at any time. I agree it's not trivial to understand, so maybe classes would be better (or even a utility that can be applied to any button?)